public/javascripts/auth-index.js

```diff
@@ -40,3 +42,30 @@
             document.querySelector('.empty-follow').style.display = 'none';
         }
     })
+     //Infinite scroll
+     const reco = document.querySelector('.reco-content');
+     const foll = document.querySelector('.follow-content');
+     const bottom = document.querySelector('.bott');
+     const bottomObserver = new IntersectionObserver( async (entries,observer) => {
+        let url = '/api/stories'
+        let follUrl = '/api/stories/following'
+        let getPosts;
+        //prevent fetch unless tab is open
+        if(reco.style.display === 'block'){
+            getPosts =  await fetch(url).then(res => res.json()).then(res => res);
+        }else if(foll.style.display === 'block'){
+            getPosts =  await fetch(follUrl).then(res => res.json()).then(res => res);
+        }
```

Comment:
It might be cleaner to use the `if` statement to determine what url to use and set it to a variable, then fetch after the `if` statement.

Also, I believe `.then(res => res);` does not need to be here.